### DISCUSSION TITLE
i3 with plasma or xfce?

### DISCUSSION THREAD

**Discussion by trangius** [i3/i3#5685] [Q&A]
So I used i3 way back and then switched to Sway because of of the naitive 2x4k without screen tearing.

Anyway, I'm considering to try out i3 on top of Plasma or Xfce, since it just seem smooth to have a proper desktop environment in the background (if I can get dual 4k without tearing). Which would you suggest to go with, Xfce or Plasma?

**Comment by mksonkar** [comment]
I'm using i3 on Plasma 6 currently. It took some tweaks to get it working right. Only thing I haven't gotten to work is highlight urgent workspace. Sometimes I click on a sign in button on an app, and it just opens the signin page on the open browser, and the workspace isn't indicated.
